source/rrRoadRunner.cpp

```diff
@@ -1639,12 +1679,19 @@ const DoubleMatrix* RoadRunner::simulate(const Dictionary* dict)
             // optimiziation for certain getValue operations.
             self.model->setIntegration(true);
 
-            int n=0;
```

Comment:
I can't see any obvious reason to change this from 0 to 1.  Can you explain?

Reply:
It is because of getSelectedValues() [here](https://github.com/sys-bio/roadrunner/blob/c82d8e7652c3512107c63a4e49bd5fb5898db76f/source/rrRoadRunner.cpp#L1672), so there is already one row in results. So the loop would stop with one row more than rr::Config::MAX_OUTPUT_ROWS.

Also, sorry for the messy commits. I had made more changes to the code later, after I had merged and found some bugs. (Accidentally changed "on windows" to "in windows" as well)

Reply:
Got it!  Thanks.